tracker/iou_tracker.py

```diff
@@ -280,13 +287,24 @@ def _associate_instances_to_tracks(
     return list_of_matches, unmatched_instances, unmatched_tracks
 
 
-def read_panoptic_image(path: Text, label_divisor: int) -> np.ndarray:
+def read_panoptic_image_2ch(path: Text, label_divisor: int) -> np.ndarray:
   """Reads in a panoptic image in 2 channel format and returns as np array."""
```

Comment:
Explain briefly what is "panoptic image in 2 channel"

Reply:
Done.